### PR TITLE
Welcome intro text + links

### PR DESCRIPTION
**Pull request by cindy1u0**
also changed the name Header to Relevant
Screen Shot 2021-01-16 at 12 30 43 PM
Screen Shot 2021-01-16 at 12 28 18 PM

### CODE REVIEW

File: src/components/Schedule.js [hunk 1]
```diff
@@ -5,62 +5,95 @@ import Arrow from "../assets/arrow.svg";
 import data from "../data/fake_data.json";
 
 const Schedule = () => {
```
Comment (cindy1u0) [inline src/components/Schedule.js:7]: hmm maybe we need some pre commit hook? I think my format ran this file again ;-;

File: src/components/Schedule.js [hunk 2]
```diff
@@ -5,62 +5,95 @@ import Arrow from "../assets/arrow.svg";
...
-                </tr>)})}
-              </tbody>
-            </Table>
-          </Tab>
-          })}
-        </Tabs>
-      </Col>
-    </Row>
-  </Container>)
+  return (
```
Comment (nupur-n) [inline src/components/Schedule.js:8]: did anything change in the schedule or was it just reformatting?

File: src/data/fake_data.json [hunk 3]
```diff
@@ -1,55 +1,55 @@
 {
-    "schedule": [
+  "schedule": [
```
Comment (nupur-n) [inline src/data/fake_data.json:2]: (same question as above)

**Review by nupur-n — APPROVED** [review]
LGTM. Just wanted to double check that the scheduling changes were all just formatting ones heh

File: src/components/Schedule.js (hunk 2 above)
Comment (cindy1u0) [reply src/components/Schedule.js:8]: ya I think they r all reformatting

File: src/data/fake_data.json (hunk 3 above)
Comment (cindy1u0) [reply src/data/fake_data.json:2]: also just reformatting which is kinda weird

**Review by snwessel — APPROVED** [review]
This looks great! Its weird that it updated the formatting, but all of the formatting updates look correct so 🤷 

Thank you for working on this section!!

**Comment by snwessel** [comment]
@cindy1u0  Feel free to merge 😄  You should be good to go!

**Pull request merged**